.readthedocs.yaml

```diff
@@ -0,0 +1,14 @@
+# [Read the Docs] configuration file
```

Comment:
can this also be inside the _config folder? or does that break anything?

Reply:
This file can only be in the repository root, or ReadTheDocs won't be able to find it. The only other option is to not have this file at all, and instead set up all configuration options through their web interface. But that's considered more fragile, so [not recommended](https://docs.readthedocs.io/en/stable/config-file/index.html).